src/main/java/org/spongepowered/api/entity/living/player/tab/TabList.java

```diff
@@ -70,45 +73,46 @@
      * @param header The new header
      * @param footer The new footer
      */
-    default void setHeaderAndFooter(Text header, Text footer) {
+    default void setHeaderAndFooter(@Nullable Text header, @Nullable Text footer) {
```

Comment:
Should mention the behaviour of the method if one of the arguments is null in the Javadocs (same for the methods above)